ui/src/app/applications/components/applications-list/applications-tiles.tsx

```diff
@@ -61,7 +61,7 @@ export const ApplicationTiles = ({applications, syncApplication, refreshApplicat
     useKeybinding(Key.UP, () => navApp(-1 * appsPerRow));
 
     useKeybinding(Key.ENTER, () => {
-        if (selectedApp > -1) {
+        if (selectedApp > -1 && (document.activeElement === document.body || document.activeElement === null)) {
```

Comment:
I think this PR is supposed to prevent navigating to an application when user select an item from autocomplete using Enter, correct? 

If this is the case then we should fix that error in autocomplete itself: autocomplete should handle enter key and stop event bubbling using `event.stopPropagation();`